rt/rs/extensions/providers/pom.xml

```diff
@@ -114,12 +114,18 @@
             <groupId>javax.json</groupId>
             <artifactId>javax.json-api</artifactId>
             <optional>true</optional>
-        </dependency>          
+        </dependency>
+        <dependency>
```

Comment:
This dependency may go in conflict with `geronimo-jsonb_1.0_spec` (https://github.com/apache/cxf/blob/CXF-7601_microProfileOpenApi/parent/pom.xml#L2213) which we would need for Geronimo OpenAPI integration ...

Reply:
No problem.  It looks like the geronimo-jsonb_1.0_spec dependency works for me too, so I'll switch to that.  Thanks for the review!